Podfile

```diff
@@ -6,11 +6,11 @@ target 'ios-showcase-template' do
   use_frameworks!
   platform :ios, '10.0'
 
-  if ENV['LOCAL']
-    pod 'AGSCore', :path => '../'
-    pod 'AGSAuth', :path => '../'
-    pod 'AGSPush', :path => '../'
-    pod 'AGSSec', :path => '../'
+  if ENV['LOCAL_DIR']
+    pod 'AGSCore', :path => ENV['LOCAL_DIR']
+    pod 'AGSAuth', :path => ENV['LOCAL_DIR']
+    pod 'AGSPush', :path => ENV['LOCAL_DIR']
+    pod 'AGSSec', :path => ENV['LOCAL_DIR']
```

Comment:
should this be `AGSSecurity`?

Reply:
It is changed in the @StephenCoady PR so I wanted to avoid conflicts, but I'm going to change it here and then we going to rebase the other PR.